src/agent/command_store/src/command_store.cpp

```diff
@@ -0,0 +1,196 @@
+#include <command_store.hpp>
+
+#include <chrono>
+#include <iostream>
+
+namespace command_store
+{
+    constexpr double MILLISECS_IN_A_SEC = 1000.0;
+
+    Status CommandStore::StatusFromInt(const int i)
+    {
+        switch (i)
+        {
+            case 0: return Status::SUCCESS; break;
+            case 1: return Status::ERROR; break;
+            case 2: return Status::IN_PROGRESS; break;
+            case 3: return Status::TIMEOUT; break;
+            default: return Status::UNKNOWN; break;
+        }
+    }
+
+    CommandStore::CommandStore()
+        : m_dataBase(std::make_unique<sqlite_manager::SQLiteManager>(COMMANDSTORE_DEFAULT_DB_PATH))
+    {
+        sqlite_manager::Column colId {"id", sqlite_manager::ColumnType::INTEGER, true, false, true};
+        sqlite_manager::Column colModule {"module", sqlite_manager::ColumnType::TEXT, true, false, false};
+        sqlite_manager::Column colCommand {"command", sqlite_manager::ColumnType::TEXT, true, false, false};
+        sqlite_manager::Column colParameter {"parameter", sqlite_manager::ColumnType::TEXT, true, false, false};
```

Comment:
```suggestion
        sqlite_manager::Column colParameter {"parameters", sqlite_manager::ColumnType::TEXT, true, false, false};
```
Fix this in all locations.